jena-maven-tools/pom.xml

```diff
@@ -20,7 +20,6 @@
   <modelVersion>4.0.0</modelVersion>
   <artifactId>jena-maven-tools</artifactId>
   <packaging>maven-plugin</packaging>
-  <version>3.7.0-SNAPSHOT</version>
```

Comment:
There have been some problems in the past which is why the top POM has this module commented out.
```
      <!-- apache-19 breaks this
          <module>jena-maven-tools</module>
        -->
```
Does jena-maven-tools now work in the build? It would be great if it did.